assets/initial_algorithms.json

```diff
@@ -0,0 +1,136 @@
+                        "ZIP": 0.95
+                    }
+                }
+            },
+            {
+                "blocking_keys": [
+                    "ZIP",
+                    "FIRST_NAME",
+                    "LAST_NAME",
+                    "SEX"
+                ],
+                "evaluators": {
+                    "ADDRESS": "func:recordlinker.linking.matchers.feature_match_fuzzy_string",
+                    "BIRTHDATE": "func:recordlinker.linking.matchers.feature_match_exact"
+                },
+                "rule": "func:recordlinker.linking.matchers.eval_perfect_match",
+                "cluster_ratio": 0.9,
+                "kwargs": {
+                    "thresholds": {
+                        "FIRST_NAME": 0.9,
+                        "LAST_NAME": 0.9,
+                        "BIRTHDATE": 0.95,
+                        "ADDRESS": 0.9,
+                        "CITY": 0.92,
+                        "ZIP": 0.95
+                    }
+                }
+            }
+        ]
+    },
+    {
+        "label": "dibbs-enhanced",
+        "description": "The DIBBs Log-Odds Algorithm. This optional algorithm uses statistical correction to adjust the links between incoming records and previously processed patients (it does so by taking advantage of the fact that some fields are more informative than others—e.g., two records matching on MRN is stronger evidence that they should be linked than if the records matched on zip code). It can be used if additional granularity in matching links is desired. However, while the DIBBs Log-Odds Algorithm can create higher-quality links, it is dependent on statistical updating and pre-calculated population analysis, which requires some work on the part of the user. For those cases where additional precision or stronger matching criteria are required, the Log-Odds algorithm is detailed below.",
+        "is_default": false,
+        "passes": [
+            {
+                "blocking_keys": [
+                    "BIRTHDATE",
+                    "MRN",
+                    "SEX"
+                ],
+                "evaluators": {
+                    "FIRST_NAME": "func:recordlinker.linking.matchers.feature_match_log_odds_fuzzy_compare",
+                    "LAST_NAME": "func:recordlinker.linking.matchers.feature_match_log_odds_fuzzy_compare"
+                },
+                "rule": "func:recordlinker.linking.matchers.eval_log_odds_cutoff",
+                "cluster_ratio": 0.9,
+                "kwargs": {
+                    "similarity_measure": "JaroWinkler",
+                    "thresholds": {
+                        "FIRST_NAME": 0.9,
+                        "LAST_NAME": 0.9,
+                        "BIRTHDATE": 0.95,
+                        "ADDRESS": 0.9,
+                        "CITY": 0.92,
+                        "ZIP": 0.95
+                    },
+                    "true_match_threshold": 12.2,
+                    "log_odds": {
```

Comment:
How will the user update these log odds in the future? Will they need to edit this configuration file directly (or pass a new algorithm in the `INITIAL_ALGORITHMS` env var)?

Reply:
There are a couple of options.  1) they could use the new `/alogirhtm*` API endpoints to create and/or update Algorithms.  2) Depending on the installation, they could use the `scripts/algorithm_config.py` CLI utility to create and/or update.  3) They could create a file very similar to this one, and update the INITIAL_ALGORITHMS env variable to point to that file.  On startup, those algorithms will be loaded.